### DISCUSSION TITLE
Date only importing as today

### DISCUSSION THREAD

**Discussion by semyag** [firefly-iii/firefly-iii#6147] [Firefly III]
First time trying to import, and no matter what role I give the column in the csv after importing the dates only show as today.  Following is a csv entry and screenshots of my import settings. 
![image](https://user-images.githubusercontent.com/65565430/172707588-f45447f3-b686-4ba9-9f9d-ec422cdfe91f.png)

![Screenshot 2022-06-08 155156](https://user-images.githubusercontent.com/65565430/172707607-54adcf44-e421-4615-8d51-5bff11f217df.png)

DEBIT,06/06/2022,"Resterauntname state                  06/05",-11.70,DEBIT_CARD,371.49,,

Am I importing wrong? How can I get the date to match the actual transaction date.

**Comment by JC5** [comment]
Change the "date format" until it matches what's in your file.

**Reply by semyag** [reply]
Thank you for the quick response! 

06/06/2022 
is an example in my CSV, now if I enter 
M/D/Y   

I get Tuesday instead of the actual date. I tried looking at the PHP date doc however its over my head a bit...

-edit: nvm didn't realize that upper and lowercase will change the results
